wrftladj/solve_em_tl.F

```diff
@@ -2782,8 +2781,8 @@ SUBROUTINE solve_em_tl ( grid , config_flags  &
                                      ips,ipe,jps,jpe,kps,kpe,          &
                                      grid%i_start(ij),grid%i_end(ij),  &
                                      grid%j_start(ij),grid%j_end(ij),  &
-                                     k_start,k_end)
-!             IF(is .ne. P_QNN)THEN
+                                     k_start,k_end)                      ! for ntu3m
+!ntu3m             IF(is .ne. P_QNN)THEN
```

Comment:
@tzuchin12 
Another example to place the comment AFTER the Fortran statement (even though the Fortran statement is commented out).

Reply:
Done.